src/Avalonia.Controls/Primitives/Thumb.cs

```diff
@@ -73,7 +73,7 @@ protected override void OnPointerMoved(PointerEventArgs e)
 
         protected override void OnPointerPressed(PointerPressedEventArgs e)
         {
-            e.Device.Capture(this);
+            e.Pointer.Capture(this);
```

Comment:
Those are probably not needed anymore since pointer gets captured and uncaptured automatically